website/templates/emails/pending_retraction_non_admin.txt.mako

```diff
@@ -0,0 +1,7 @@
+Hello ${user.fullname},
+
+I just wanted to let you know, ${initiated_by} has requested a retraction for the following registration: ${registration_link}.
+
```

Comment:
To your aside: I followed a similar thought process. Either way we should agree on the behavior before I run and spend a few hours changing this behavior. 

To reiterate the idea is: Should the user initiating a Retraction or Embargo also receive an email? Furthermore, does the action of initiating the Retraction or Embargo imply their approval?

@sloria @JeffSpies thoughts?